src/main/java/org/folio/rest/utils/PostgresClientUtil.java

```diff
@@ -22,8 +22,7 @@ private PostgresClientUtil() {
    * @return the PostgresClient instance for the vertx and the tenantId
    */
   public static PostgresClient getInstance(Context vertxContext, Map<String, String> okapiHeaders) {
-    PostgresClient postgresClient = PostgresClient.getInstance(
+    return PostgresClient.getInstance(
         vertxContext.owner(), TenantTool.tenantId(okapiHeaders));
-    return postgresClient;
```

Comment:
The change is ok, but I would rather prefer to drop PostgresClientUtil and use PgUtil.postgresClient instead.